src/spec/vk.xml

```diff
@@ -521,8 +521,8 @@ maintained in the master branch of the Khronos Vulkan Github project.
             <member optional="true"><type>uint32_t</type>               <name>enabledExtensionCount</name></member>
             <member len="enabledExtensionCount,null-terminated">const <type>char</type>* const*      <name>ppEnabledExtensionNames</name></member>        <!-- Extension names to be enabled -->
             <validity>
-                <usage>Any given element of pname:ppEnabledLayerNames must: be the name of a layer present on the system, exactly matching a string returned in the sname:VkLayerProperties structure by fname:vkEnumerateInstanceLayerProperties</usage>
-                <usage>Any given element of pname:ppEnabledExtensionNames must: be the name of an extension present on the system, exactly matching a string returned in the sname:VkExtensionProperties structure by fname:vkEnumerateInstanceExtensionProperties</usage>
+                <usage>Each layer name in the pname:ppEnabledLayerNames array must: exactly match a string returned in the sname:VkLayerProperties structure from flink:vkEnumerateInstanceLayerProperties</usage>
+                <usage>Each extension name in the pname:ppEnabledExtensionNames array must: be one of those provided by the implementation or by an enabled layer and must exactly match a string returned in the sname:VkExtensionProperties structure from flink:vkEnumerateInstanceExtensionProperties</usage>
                 <usage>If an extension listed in pname:ppEnabledExtensionNames is provided as part of a layer, then both the layer and extension must: be enabled to enable that extension</usage>
```

Comment:
Right. We've kind of drifted away from this.

Yes, with the modified "Each extension name ... or by an enabled layer ... " covers the "If an extension ..." line.